libraries/botbuilder/tests/teamsInfo.test.js

```diff
@@ -935,13 +954,6 @@ describe('TeamsInfo', function () {
                     new Error('This method requires a connector client.')
                 );
             });
-
-            it(`should error if the adapter doesn't have a createConnectorClient method`, function () {

```

Comment:
This should probably verify that a context without a connector client still throws.

Reply:
[This](https://github.com/microsoft/botbuilder-js/pull/3830/commits/902a1d53bf52604fe317e9cb9b4d5fc5b907fb03) work? Note: The test above this covers "should error if the context doesn't have an adapter".

Reply:
Perfect!